include/alpaka/core/CudaHipMath.hpp

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2019-2021 Benjamin Worpitz, René Widera, Sergei Bastrakov
+ *
+ * This file is part of alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#include <alpaka/core/BoostPredef.hpp>
+#include <alpaka/core/UniformCudaHip.hpp>
+
+#if defined(ALPAKA_ACC_GPU_CUDA_ENABLED)
+#    include <cuda_runtime.h>
+#endif
+
+#if defined(ALPAKA_ACC_GPU_HIP_ENABLED)
+#    if BOOST_COMP_NVCC >= BOOST_VERSION_NUMBER(9, 0, 0)
+#        include <cuda_runtime_api.h>
+#    else
```

Comment:
Didn't we drop targetting CUDA via HIP? Because then we can remove this branch.

Reply:
well spotted, yes CUDA support via hip was removed with #1338. This workaround can be removed.

Reply:
Thanks! Will remove then

Reply:
I guess we can replace the entire block by:
```
#if defined(ALPAKA_ACC_GPU_HIP_ENABLED)
#    include <hip/math_functions.h>
#endif
```

Reply:
Okay, so then there was not much point in moving it all into a separate header. But i guess it doesn't hurt anyways, just doesn't help as much as I originally thought.

Reply:
There must be a branch for CUDA and one for HIP.